### DISCUSSION TITLE
log contains errors

### DISCUSSION THREAD

**Discussion by helgator** [borgbase/vorta#1933] [General]
What does this tell me?

2024-02-12 12:47:06,177 - vorta.borg.borg_job - WARNING - ID: fbc717900868e12ecb7ff4a93156faedfcd821ce98b4997d0b63a7e65edcf7ec rebuilt index: <not found>      committed index: (3, 127366685) 

Thanks

**Comment by real-yfprojects** [comment]
Can you provide more context about your setup, the steps leading to the warning and a larger excerpt of the logs?

**Comment by helgator** [comment]
This is the log:
[vorta.log](https://github.com/borgbase/vorta/files/14254233/vorta.log)
I have Ubuntu 22.04. The backup runs automatically daily on a separate partition of the internal SSD. After Vorta is finished I get a message that I need to look into the log but I cannot make sense of the entries. 
Thanks.

**Reply by real-yfprojects** [reply]
It seems like parts of your repository are corrupted. Luckily borg is usually able to recover from that.
Try running the following sequence of commands. Note that it is [recommended](https://borgbackup.readthedocs.io/en/stable/faq.html#why-does-running-borg-check-repair-warn-about-data-loss) to do a copy of the repo beforehand.

```
borg check --info --repair {repo}
borg check --info {repo}
```
